components/CalendarCard/CalendarCard.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { storiesOf } from '@storybook/react'
+import { CalendarCard } from './CalendarCard'
+
+const data = [
```

Comment:
Did you use all of the records? Should only declare what is actually being used.